config.yaml

```
@@ -17,10 +17,10 @@ streams:
     env:
       COSA_TESTISO_DEBUG: true
     osbuild_experimental: true
-  # branched:
-  #  type: mechanical
-  #  env:
-  #    COSA_USE_OSBUILD: true
+  branched:
+    type: mechanical
+    env:
+      COSA_USE_OSBUILD: true
```

This is old.. let's replace it with:

```suggestion
    osbuild_experimental: true
```

---

done, I have made the changes